data-science-onramp/data-cleaning/requirements.txt

```diff
@@ -0,0 +1,6 @@
+grpcio==1.29.0
+google-auth==1.16.0
+google-auth-httplib2==0.0.3
+google-cloud==0.34.0
+google-cloud-storage==1.28.1
+google-cloud-dataproc==0.8.0
```

Comment:
These two packages should be the only dependencies you need to specify to get storage and dataproc.

`grpcio` and `google-auth` are pulled in as transitive dependencies, `google-auth-httplib2` is only needed for `google-api-python-client`, and `google-cloud` is a deprecated package.